relayer/src/worker/client.rs

```diff
@@ -52,40 +58,47 @@ pub fn detect_misbehavior_task<ChainA: ChainHandle, ChainB: ChainHandle>(
 ) -> Option<TaskHandle> {
     if client.is_expired_or_frozen() {
         warn!(
-            "skipping detect misbehavior task on frozen client: {}",
-            client.id()
+            client = %client.id(),
+            "skipping detect misbehavior task on frozen client",
         );
         return None;
     }
 
     {
-        debug!("[{}] doing first misbehavior check", client);
+        let _span = span!(
+            tracing::Level::DEBUG,
+            "DetectMisbehaviorFirstCheck",
+            client = %client.id,
+            src_chain = %client.src_chain.id(),
+            dst_chain = %client.dst_chain.id(),
+        )
```

Comment:
This could be folded into the "DetectMisbehaviorWorker" span below, but I thought it would be useful to distinguish these contexts since the other one is performed in a background worker task.